marathon

```diff
@@ -2,16 +2,19 @@
 # See documentation and license in README.md
 
 if [[ $1 == "--toggle" ]]; then
-	command=$2
+	shift
+	toggle=1
 else
-	command=$1
+	toggle=0
 fi
+
+command="$*"
 # zenity --info --text "$command"; # useful for debug
 
 runningInstancesOfExec=$(ps -a -x -o command | grep -v grep | grep -v defunct | grep -v marathon | grep -c $command);
 
 if [ $runningInstancesOfExec -gt 0 ]; then
-	if [[ $1 == "--toggle" ]]; then
+	if [[ toggle == "1" ]]; then
```

Comment:
@stek29 typo: `toggle` should be `$toggle`. By the way, what does your patch enable? (What's your use case where you'd call marathon with multiple arguments?)

EDIT: I'd guess you're having multiple sessions of the same app, e.g. `gvim -S work` and `gvim -S home`, and you want a keystroke per app even if it's the same binary. Right?